docs/source/optim.rst

```diff
@@ -146,6 +146,38 @@ allows dynamic learning rate reducing based on some validation measurements.
 Learning rate scheduling should be applied after optimizer's update; e.g., you
 should write your code this way:
 
+Example::
+
+    scheduler = ExponentialLR(optimizer, gamma=0.9)
+    for epoch in range(20):
+        for input, target in dataset:
+            optimizer.zero_grad()
+            output = model(input)
+            loss = loss_fn(output, target)
+            loss.backward()
+            optimizer.step()
+            scheduler.step()
```

Comment:
thanks for adding this code example! i like that it emphasize that scheduler step is done after the optimizer step.

Reply:
nit: though some schedulers are applied only at every epoch. do you have a way of suggesting this without adding too much here? if it's cumbersome, we don't need to make that point :)

Reply:
Thanks for the amazing review :) and i did a little simplification. Hope you're liking the current version.